.github/workflows/build-git-installers.yml

```diff
@@ -206,15 +206,10 @@ jobs:
         run: |
           set -x
 
-          b=/usr/src/build-extra &&
-          str="recently_seen=.*\$" &&
-
-          echo $b/git-extra/git-update-git-for-windows &&
-          cat $b/git-extra/git-update-git-for-windows &&
+          b=/mingw64/bin &&
 
-          sed -i -e '/^# when confirmation to do so is given./a\
-            use_recently_seen=no' \
-            $b/git-extra/git-update-git-for-windows
+          sed -i -e '6 a use_recently_seen=no' \
```

Comment:
I would prefer the form that does not expect line number 6 to be the one, but that matches the line after which we want to append.

Reply:
The good news is that this area of the file is all comments, so inserting to a specific line is more stable than matching the text of a comment, hence the choice I made here.